zenoh_c_vendor/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ set(ZENOHC_CARGO_FLAGS "--no-default-features$<SEMICOLON>--features=zenoh/shared
 # - https://github.com/eclipse-zenoh/zenoh/pull/1150 (fix deadlock issue https://github.com/ros2/rmw_zenoh/issues/182)
 ament_vendor(zenoh_c_vendor
   VCS_URL https://github.com/eclipse-zenoh/zenoh-c.git
-  VCS_VERSION 548ee8dde0f53a58c06e68a2949949b31140c36c
+  VCS_VERSION 134dbfa06ca212def5fb51dd8e816734dfd8dff6
```

Comment:
With #269 merged, the commit version is the latest `zenoh-c` so we don't have to change the commit here to an older one.